lib/livingstyleguide/integration/rails.rb

```diff
@@ -3,7 +3,9 @@
   require 'rails'
   class LivingStyleGuideRailtie < Rails::Railtie
     initializer 'living_style_guide.assets' do
-      Rails.application.assets.register_engine('.lsg', ::LivingStyleGuide::TiltTemplate)
+      Rails.application.config.assets.configure do |env|
+        env.register_engine('.lsg', ::LivingStyleGuide::TiltTemplate)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.